releases/releaser/releaser.go

```diff
@@ -198,7 +185,8 @@ func (r *Releaser) uploadChecksums(ctx context.Context, params PublishParameters
 }
 
 func (r *Releaser) walkFiles(basePath string, includeFn func(path string, info os.FileInfo) bool,
-	observerFn func(path string, info os.FileInfo) error) error {
+	observerFn func(path string, info os.FileInfo) error,
```

Comment:
This goes against the style we use for multiline functions, please back out this change.